resources/translations/es_BO.ts

```diff
@@ -6,7 +6,7 @@
     <message>
         <location filename="../qml/pages/AboutPage.qml" line="13"/>
         <source>Shmoose - Xmpp Client</source>
-        <translation>Shmoose un cliente de XMPP</translation>
+        <translation>Shmoose un cliente no oficial de XMPP</translation>
```

Comment:
official when is coming from original vendor, not official when is coming from not vendor, but I am translating  I lot of sailfish app,  which I made some mistakes, that I correct them eventually, also translating spanish is not easy because every country speak spanish they interpret the words in their own way, which could not be very consistent with what I do. Feel free to for commenting your opinion on it.
Thanks

Reply:
ok I did a  short revision on it then in my guidelines,styles,rules, I call ¨no official¨ to ¨client¨ and ¨official ¨ to ¨native¨.  So it is the same in the end, It is just my rules, but I can change this if you want. Just let me know geobra.
Thansk.

Reply:
What is 'original vendor' in your opinion? Jolla? So every app that is not from jolla is a 'no official' app? Or is it xmpp? There is no official xmpp app. Xmpp is a standard. Like E-Mail. There is no 'official' E-Mail app. So, I don't get it what you mean with 'no official'...

Reply:
not jolla, not sailfish, or whatever. I labeled "no oficial" because is not coming from in this case "xmpp.org".  also "no oficial" is an extra comment to be more specific. But I say before, I fix errors eventually.